This repository contains the working data and code that was used in the paper "Fungal Endophytes in the Sierra Nevada Foothills: Comparing and contrasting foliar fungal endophyte communities in blue oaks and interior live oaks" by Christopher Bivins and Carolin Frank

This project involved the use of fungal ITS metabarcoding using an Illumina MiSeq 250 Paired-End sequencing approach to determine how fungal endophyte communities are structured in Quercus douglasii (blue oak) and Quercus wislizeni (interior live oak) leaves. Our sampling process involved taking 4 leaves per tree, surface sterilizing them, grinding, and pooling four leaves per sample, followed by DNA extraction with the Qiagen DNEASY POWERSOIL PRO KIT, and then PCR with Illumina adapted ITS1 and ITS2 primers. We sampled from 48 individual trees (24 blue oaks and 24 interior live oaks). 12 blue oaks and 12 interior live oaks were sampled from a ridge at our site, which was hot, dry, and exposed. We also sampled 12 blue oaks and 12 interior live oaks from a valley at our site, which was cool, moist, and shady. We sampled the same 48 trees three times: once in May, once in July, and once in September. The goal was to understand how fungal endophyte communities vary between both oak species, between sites, and over time.  

"Working data" refers to data that has already been processed for analysis. Code for all sequence processing as well as code for all analyses is available in this repository. See the paper for detailed information on our methods. 

Raw read data, in the form of fastq files, has been accessioned to GenBank and can be found under the following BioProject ID: PRJNA1212509

For any questions or comments, please contact the corresponding author, Christopher Bivins, at cbivins@ucmerced.edu
